src/NebulaServiceProvider.php

```diff
@@ -19,6 +19,7 @@ public function boot(): void
     {
         $this->loadJsonTranslationsFrom(__DIR__.'/../resources/lang', 'nebula');
         $this->loadViewsFrom(__DIR__.'/../resources/views', 'nebula');
+        $this->loadViewsFrom(resource_path('views/vendor/nebula'), 'nebula');
```

Comment:
Is there a reason why you use `resource_path` instead of `__DIR__`? I prefer `__DIR__` due to consistency

Reply:
This will load from the app views folder, if I use `__DIR__` then it'll be like `__DIR__.'/../../../../../views/vendor/nebula'` which looks odd also `resource_path` always point to resource directory, where as if I use `__DIR__` and you later move the `NebulaServiceProvider` in different directory (which is unlike I know) then this path have to be updated.
But if you still want to use `__DIR__` let me know I'll update the PR

Reply:
Admittedly, this bit of code isn't actually needed. Since the views are being namespaced, Laravel will automatically check the `views/vendor/nebular` folder when using `nebula::` views.